frontend/src/components/userPage/CreateEventForm.jsx

```diff
@@ -0,0 +1,216 @@
+                        >
+                        Name:
+                        </label>
+                        <input
+                        type="text"
+                        className="border rounded w-full py-2 px-3 text-gray-700 leading-tight"
+                        value={name}
+                        onChange={(e) => setEventName(e.target.value)}
+                        required
+                        />
+                    </div>
+
+                    <div className="mb-4">
+                        <label
+                        className="block text-gray-800 text-sm font-bold mb-2"
+                        >
+                        Category:
+                        </label>
+                        <input
+                        type="text"
+                        className="border rounded w-full py-2 px-3 text-gray-700 leading-tight"
+                        value={category}
+                        onChange={(e) => setCategory(e.target.value)}
+                        required
+                        />
+                    </div>
+
+                    <div className="mb-4">
+                        <label
+                        className="block text-gray-800 text-sm font-bold mb-2"
+                        >
+                        Location:
+                        </label>
+                        <input
+                        type="text"
+                        className="border rounded w-full py-2 px-3 text-gray-700 leading-tight"
+                        value={eventLocation}
+                        onChange={(e) => setEventLocation(e.target.value)}
+                        required
+                        />
+                    </div>
+
+                    <div className="mb-4">
+                        <label
+                        className="block text-gray-800 text-sm font-bold mb-2"
+                        >
+                        Latitude:
+                        </label>
+                        <input
+                        type="text"
+                        className="border rounded w-full py-2 px-3 text-gray-700 leading-tight"
+                        value={lat}
+                        onChange={(e) => setLat(e.target.value)}
+                        required
+                        />
+                    </div>
+
+                    <div className="mb-4">
+                        <label
+                        className="block text-gray-800 text-sm font-bold mb-2"
+                        >
+                        Longitude:
+                        </label>
+                        <input
+                        type="text"
+                        className="border rounded w-full py-2 px-3 text-gray-700 leading-tight"
+                        value={long}
+                        onChange={(e) => setLong(e.target.value)}
+                        required
+                        />
+                    </div>
```

Comment:
So user needs to manually fill in latitude and longitude?

Reply:
Currently, yes.
**_To Do:_**
Implement a system to get long and lat